requirements.txt

```diff
@@ -1,14 +1,22 @@
-fastapi==0.67.0
-fastapi_utils==0.2.1
-gunicorn==20.1.0
-numpy==1.21.1
+numpy==1.20.3
+requests==2.25.1
+pydantic==1.7.4
 psycopg2-binary==2.9.1
```

Comment:
We need to go through and delete all lines involving psycopg2, it has been successfully replaced (great work by the way) and is no longer in use. Anywhere I see it written is commented out.

Reply:
This. Leave no traces of psycopg2.